lib/AST/NameLookup.cpp

```diff
@@ -1364,6 +1365,11 @@ TinyPtrVector<ValueDecl *> NominalTypeDecl::lookupDirect(
   bool useNamedLazyMemberLoading = (ctx.LangOpts.NamedLazyMemberLoading &&
                                     hasLazyMembers());
 
+  if (getBaseName().getIdentifier().str().equals("NotificationCenter") &&
+      name.getBaseName().getKind() == DeclBaseName::Kind::Normal &&
+      name.getBaseName().getIdentifier().str().equals("addObserver"))
+    fprintf(stderr, "---NotificatiobCenter.addObserver\n");
```

Comment:
Debugging code left here

Reply:
I removed it from a follow-up commit, thank you!